src/trix/config/block_attributes.coffee

```diff
@@ -5,6 +5,10 @@ Trix.config.blockAttributes = attributes =
   quote:
     tagName: "blockquote"
     nestable: true
+  heading:
```

Comment:
+1 for header-1 ... header-6. I know it's not as semantic as @javan's observation, but I like the close mapping to HTML. I'd rather not have to tweak the attribute names to match a more application specific name.